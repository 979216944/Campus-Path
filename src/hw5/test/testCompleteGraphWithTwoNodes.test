# Test that two nodes and four edges can be added.(complete graph)

# Create a graph
CreateGraph g

# Add two nodes
AddNode g a
AddNode g b

# Add four edges
AddEdge g a a aa
AddEdge g b b bb
AddEdge g a b ab
AddEdge g b a ba

# Print nodes and the outgoing edges from a and b
ListNodes g
ListChildren g a
ListChildren g b