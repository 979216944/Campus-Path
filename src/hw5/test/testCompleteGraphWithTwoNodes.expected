# Test that two nodes and four edges can be added.(complete graph)

# Create a graph
created graph g

# Add two nodes
added node a to g
added node b to g

# Add four edges
added edge aa from a to a in g
added edge bb from b to b in g
added edge ab from a to b in g
added edge ba from b to a in g

# Print nodes and the outgoing edges from a and b
g contains: a b
the children of a in g are: a(aa) b(ab)
the children of b in g are: a(ba) b(bb)